content/en/docs/tasks/administer-cluster/verify-signed-artifacts.md

```diff
@@ -116,3 +116,29 @@ Here are some helpful resources to get started with `policy-controller`:
 
 - [Installation](https://github.com/sigstore/helm-charts/tree/main/charts/policy-controller)
 - [Configuration Options](https://github.com/sigstore/policy-controller/tree/main/config)
+
+## Verify the Software Bill Of Materials (SBOM)
+
+The Kubernetes SBOM is signed as well and can be verified by using the sigstore
+certificate and signature or the corresponding SHA files:
+
+```shell
+VERSION=$(curl -Ls https://dl.k8s.io/release/stable.txt)
+
+# SHA512 verification
+curl -Ls "https://sbom.k8s.io/$VERSION/release" -o "$VERSION.spdx"
```

Comment:
I think we should add some text description about what each command is doing.

Reply:
Yep, done :+1: